doc/operator/grafana/grafana.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: grafana
+  namespace: observatorium
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: grafana
+  template:
+    metadata:
+      name: grafana
+      labels:
+        app: grafana
+    spec:
+      containers:
+      - name: grafana
+        image: grafana/grafana:latest
+        ports:
+        - name: grafana
+          containerPort: 3000
+        resources:
+          limits:
+            memory: "2Gi"
+            cpu: "1000m"
+          requests: 
+            memory: "1Gi"
```

Comment:
that's pretty huge, I don't think Grafana needs this much baseline

Reply:
what would be reasonable here?

Reply:
I run Grafana with which is plenty:
```
    Limits:
      cpu:     200m
      memory:  200Mi
    Requests:
      cpu:        100m
      memory:     100Mi
```

Cluster monitoring operator only specifies the request:
https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/grafana/deployment.yaml#L27-L30

Reply:
Done